src/lib/cadastre/mun/mun-selector/mun-selector.component.ts

```diff
@@ -0,0 +1,124 @@
+import {
+  Component,
+  Input,
+  Output,
+  EventEmitter,
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  OnInit,
+  OnDestroy, NgModule
+} from '@angular/core';
+
+import {BehaviorSubject, Subject, Subscription} from 'rxjs';
+
+import {
+  EntityStore,
+  EntityStoreController,
+  EntityRecord
+} from '@igo2/common';
+
+import { MunNom } from 'src/lib/cadastre/mun/shared/mun.interfaces';
+import {MunService} from 'src/lib/cadastre/mun/shared/mun.service';
```

Comment:
Namespace MunService with Cadastre, meaning MunService should be CadastreMunService.